Duckling/Numeral/EN/Rules.hs

```diff
@@ -352,6 +352,18 @@ ruleLegalParentheses = Rule
     _ -> Nothing
   }
 
+-- number1., Number1
```

Comment:
I would actually think this to be an ordinal, not a numeral. "Number 1" would correspond to "the 1st thing". For example, to say "They are the number one player in the world". This means "they" are 1st. Or, "I would like to order a number one". Again, 1st. However in English it is much more rare to say "number one" to mean the actual number, 1, unless preceded by the article "the". For example, "the number one". But even then it's context dependent, and this could also be an ordinal.